include/QuantumComputation.hpp

```diff
@@ -49,6 +50,9 @@ namespace qc {
         ClassicalRegisterMap cregs{};
         QuantumRegisterMap   ancregs{};
 
+        std::mt19937_64 mt;
+        std::size_t     seed = 0;
```

Comment:
Make this const?

Reply:
I'd rather keep it non-const so that the defaulted move-assignment still works as it is.